Week2/MohammadDarwesh/mohammaddarwesh.js

```diff
@@ -0,0 +1,180 @@
+// for the first task i will do it with many ways just for practising.
+console.log ("Halo, dunia! ,Ciao, mondo! ,Hola, mundo!");
+
+let indonesian = "Halo, dunia!";
+let italian = "Ciao, mondo!";
+let spanish = "Hola, mundo!";
+console.log(indonesian ,italian, spanish );
+
+let helloWorld = [`${indonesian}` ,`${italian}` ,`${spanish}`];
+console.log(helloWorld);
+console.log(helloWorld[0]);
+console.log(helloWorld[1]);
+console.log(helloWorld[2]);
+
+let helloWorld2 = ["Halo, dunia!" ,"Ciao, mondo!" ,"Hola, mundo!"];
+console.log(helloWorld2);
+console.log(helloWorld2[0]);
+console.log(helloWorld2[1]);
+console.log(helloWorld2[2]);
+console.log(helloWorld2.join("-"));
+console.log(helloWorld2.reverse());
+
+let helloWorld3 = {
+    indonesian: "Halo, dunia!",
+    italian: "Ciao, mondo!",
+    spanish: "Hola, mundo!"
+   };
+console.log(helloWorld3);
+console.log(helloWorld3.indonesian);
+console.log(helloWorld3.italian);
+console.log(helloWorld3.spanish);
+// end of task 1.
+
+
+// task 2:
+console.log("I'm awesome");
```

Comment:
Just for your information, you can also use escape characters(using backslash) like this:
`console.log('I\'m awesome');`

The middle single quote will be then interpreted by JS as a string character and not a special character.

Reply:
Thank you for your feed back .
I know already about (backslash).... but in general most of the questions are not specific....i think it will be better if the question is (can you find many ways to solve the error? ).